### PR TITLE
CAL-373: Added OWASP Suppression for Solr 6.6.1 CVE-2017-12629

### PR DESCRIPTION
**Pull request by figliold**
What does this PR do?

Adds Suppression for Solr 6.6.1 CVE-2017-12629.

Who is reviewing it?

@clockard

Select relevant component teams:

https://github.com/orgs/codice/teams

Choose 2 committers to review/merge the PR.

@lessarderic
@coyotesqrl

How should this be tested? (List steps with links to updated documentation)

Run OWASP

Any background context you want to provide?

What are the relevant tickets?

https://codice.atlassian.net/browse/CAL-373

Screenshots (if appropriate)

Checklist:

Documentation Updated
Update / Add Unit Tests
Update / Add Integration Tests
Review Comment Legend:

✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by figliold** [comment]
build now

**Comment by clockard** [comment]
build now

File: dependency-check-maven-config.xml [hunk 1]
```diff
@@ -542,4 +542,12 @@
         </notes>
         <cve>CVE-2016-8739</cve>
     </suppress>
-</suppressions>
\ No newline at end of file
+    <!-- Remove this suppression when DDF-3440 is worked -->
```
Comment (coyotesqrl) [inline dependency-check-maven-config.xml]: Approving, but we should include a link to this block in the Jira ticket so it is not lost.

Comment (figliold) [reply dependency-check-maven-config.xml]: done

**Pull request merged**